### ISSUE TITLE
Use embedded playground in the tutorial of "HTML5 Video"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#465]
```
Hi Paul,
I'd like to invite you to try embedded playground in the your tutorial of  
"HTML5 Video", esp. the examples in "2. Video Formats" and "5. The Fun Part" 
sections.

The tutorial and help doc links are listed below:

Tutorial Link:
http://www.html5rocks.com/tutorials/video/basics/

Guide on how to use embedded playground:
https://docs.google.com/a/google.com/document/d/1ReJCAeh9wW-JO07Wx7uWferqg3BQwMP
dubORobNkQwU/edit?hl=en#

Feel free to let me know if you have any questions.

Thanks!

-hanrui
```

Original issue reported on code.google.com by `han...@google.com` on 25 Feb 2011 at 7:21

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Sorry, not Paul, I mean Ernest...
Thanks!
```

Original comment by `han...@google.com` on 25 Feb 2011 at 7:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 11 Apr 2011 at 4:36
- Added labels: **Milestone-6**
- Removed labels: **Milestone-5.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Aug 2011 at 6:04
- Added labels: **Milestone-7.1**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 2a576c348cbc.
Ernest, could you please take a look? Thank you!

```

Original comment by `han...@google.com` on 7 Sep 2011 at 10:02
- Changed state: **Fixed**
- Added labels: **Milestone-7.2**
- Removed labels: **Milestone-7.1**
